### PR TITLE
[security]: update kernel to 4.19.152

### PR DESCRIPTION
**Pull request by lguohan**
security changelog from

https://salsa.debian.org/kernel-team/linux/-/raw/buster-security/debian/changelog

remove e1000-Do-not-perform-reset-in-reset_task-if-we-are-a.patch as
it is included in the debian upstream kernel

Signed-off-by: Guohan Lu <lguohan@gmail.com>

### CODE REVIEW

**Review by qiluo-msft — APPROVED** [review]

File: patch/Support-for-fullcone-nat.patch [hunk 1]
```diff
@@ -1,6 +1,7 @@
 From 67cc2bd46c7ad1431ddd1819cdfad6ecc08a7593 Mon Sep 17 00:00:00 2001
+
```
Comment (paulmenzel) [inline patch/Support-for-fullcone-nat.patch:2]: I’d leave it how `git format-patch` generates it.

File: patch/Support-for-fullcone-nat.patch [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
 From: Kiran Kella <kiran.kella@broadcom.com>
-Date: Fri, 6 Sep 2019 20:54:19 -0700
```
Comment (paulmenzel) [inline patch/Support-for-fullcone-nat.patch:3]: I’d leave the date, as a Date: field is required.

**Comment by paulmenzel** [comment]
How did you rebase the patches? Use some quilt tool or just in a git repository?

**Comment by lguohan** [comment]
@paulmenzel , i used stg.

File: patch/Support-for-fullcone-nat.patch (hunk 1 above)
Comment (lguohan) [reply patch/Support-for-fullcone-nat.patch:2]: all patches are generated by stg

**Pull request merged**